### PR TITLE
12.0 mig partner_external_map

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa

### CODE REVIEW

File: partner_external_map/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    'summary': 'Add Map and Map Routing buttons on partner form to '
+               'open GMaps, OSM, Bing and others',
+    'author': 'Akretion, '
+              'Tecnativa, '
+              'Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/partner-contact',
+    'depends': [
+        'base',
+    ],
+    'data': [
```
Comment (nikul-serpentcs) [inline partner_external_map/__manifest__.py:20]: @ernestotejeda Improve Data sequence should be [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L29)

**Review by nikul-serpentcs — COMMENTED** [review]
Minor Change

File: partner_external_map/__manifest__.py (hunk 1 above)
Comment (pedrobaeza) [reply partner_external_map/__manifest__.py:20]: @nikul-serpentcs I wouldn't enforce that order as not always can be done that way due to XML-ID dependencies. There's no guideline about that either.

**Review by pedrobaeza — APPROVED** [review]

File: partner_external_map/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -0,0 +1,6 @@
+* `Akretion <http://www.akretion.com>`_:
+  * Alexis de Lattre <alexis.delattre@akretion.com>
```
Comment (pedrobaeza) [inline partner_external_map/readme/CONTRIBUTORS.rst:3]: You should leave line breaks between this for proper rendering.

File: partner_external_map/__manifest__.py (hunk 1 above)
Comment (nikul-serpentcs) [reply partner_external_map/__manifest__.py:20]: Yes @pedrobaeza , But here `security` file is last, it's must be first

And we will try to make good sequence

**Comment by pedrobaeza** [comment]
Can you add an address with all fields in the street for full module test coverage?

File: partner_external_map/__manifest__.py (hunk 1 above)
Comment (pedrobaeza) [reply partner_external_map/__manifest__.py:20]: Why? It doesn't matter. It's going to work exactly the same.

Comment (nikul-serpentcs) [reply partner_external_map/__manifest__.py:20]: @pedrobaeza Because `security` files  it's must be first not last

Comment (pedrobaeza) [reply partner_external_map/__manifest__.py:20]: No, where do you find that requirement?

Comment (nikul-serpentcs) [reply partner_external_map/__manifest__.py:20]: @pedrobaeza  [here]( https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L30)

but, Here not security XML file then It's Fine

Comment (pedrobaeza) [reply partner_external_map/__manifest__.py:20]: No, that's a template where the order was not intended. Only an accident.

**Pull request merged**